02.calendar/calendar.js

```diff
@@ -0,0 +1,22 @@
+import minimist from "minimist";
+const argv = minimist(process.argv.slice(2), {
+  alias: {
+    m: "month",
+    y: "year",
+  },
+});
+let now = new Date();
+let year = argv.year || now.getFullYear();
+let month = argv.month || now.getMonth() + 1;
+let day_first = new Date(year, month - 1, 1);
+let day_end = new Date(year, month, 0);
+console.log(year + "年" + month + "月");
+console.log(" 日 月 火 水 木 金 土");
```

Comment:
OS標準のcalコマンドだとスペースは入らないのでは？
```suggestion
console.log("日 月 火 水 木 金 土");
```

Reply:
確認ミスでした。ご指摘ありがとうございます。